package.json

```diff
@@ -5,7 +5,7 @@
   "author": "Rapid Silicon",
   "main": "main.js",
   "scripts": {
-    "start": "webpack --mode development && electron .",
+    "start": "webpack --mode development && NODE_ENV=development electron .",
```

Comment:
Is the dev mode temporary?

Reply:
dev mode is for daily development. For production it must be something else.